app/test.jsx

```diff
@@ -251,9 +251,10 @@ class App extends Component {
             <ChartTitle
                 onChange={this.symbolChange}
                 active_category={this.state.activeCategory}
```

Comment:
should we remove this line since we don't use `activeCategory` from state anymore?

Reply:
@balakrishna-binary just fixed it